src/views/Receiver.vue

```diff
@@ -382,6 +394,8 @@ export default {
     setVolume(event) {
       this.$refs.audioPlayer.volume = event.srcElement.valueAsNumber / 100;
       this.$refs.videoPlayer.volume = event.srcElement.valueAsNumber / 100;
+      localStorage.setItem('volume_num', event.srcElement.valueAsNumber / 100);
+      this.volumeNum = event.srcElement.valueAsNumber / 100;
```

Comment:
I don't believe `volumeNum` is defined in the component, so this line isn't doing anything. (thanks @codysherman)

I'm surprised this is still working, since the `.volume` data property is bound to the value of the slider, so I would think the volume slider wouldn't let the user change it. However, it seems like it's working. This should be changed to `this.volume = ...` and it shouldn't break anything and make more sense.